installer/src/components/AccountForm/AccountForm.tsx

```diff
@@ -0,0 +1,147 @@
+    e.preventDefault();
+    setIsLoading(true);
+    if (accountSid && accountAuthToken) {
+      // This call creates the workspace and routers via APIs
+      await fetch(getBackendUri() + "/task-router/deploy-task-router")
+        .then(resp => resp.json())
+        .then(data => {
+          console.log(data);
+        })
+        .catch(err => console.error(err));
+
+      // This call gets all the plugins
+      const plugin = await fetch(getBackendUri() + "/plugin/get-plugins")
+        .then(plugin => plugin.json())
+        .then(async resp => {
+          const plugins: Plugin[] = resp.data.plugins;
+          const hlsPlugin = Object.values(plugins).find(plugin => plugin.friendly_name === hlsPluginName);
+          if (hlsPlugin) {
+            setPluginFriendlyName(hlsPlugin.friendly_name);
+            setPluginSid(hlsPlugin.sid);
+          }
+          return hlsPlugin;
+        })
+        .catch(err => console.error(err));
+      if (!plugin) setHlsPluginExists(false);
+      
+      // Release the plugin config to Flex Account
+      const configRelease = await fetch(getBackendUri() + "/plugin/release-plugin")
+        .then(res => res.json())
+        .then(release => release.data)
+        .catch(err => console.error(err));
+      if (configRelease) setIsLoading(false);
+      console.log(configRelease);
+    }
+  }
+
+  useEffect(() => {
+    const getAccount = async () => {
+      await fetch(getBackendUri() + "/account/get-account-name")
+      .then(resp => resp.json())
+      .then(jsonResp => setAccountName(jsonResp.data.accountName));
+
+      await fetch(getBackendUri() + "/account/get-account-info")
+        .then(resp => resp.json())
+        .then(async (jsonResp) => {
+          setAccountSid(jsonResp.data.accountSid);
+          setAccountAuthToken(jsonResp.data.authToken);
+      });
+    }
+    getAccount();
+  }, []);
+
+  return (
+    <div className='col-start-5 col-span-4 flex flex-col'>
+      <form className=''>
+        <div className='flex flex-col'>
+          <div className='row mb-5 text-deploy-blue'>
+              <div className='mb-3'>
+                <span className='font-bold pr-2'>Step 1:</span>
+                <span>Account Information</span>
+              </div>
+              <div className='pl-4'>
+                <div className='p-0'>
+                  <div className='mb-2'>Account Name</div>
+                  <input value={accountName} disabled className='bg-light-gray px-2 py-2 rounded-md border'></input>
+                </div>
+              </div>
+          </div>
+          
+          <div className='row mb-5 text-deploy-blue'>
+            <div className='mb-3 items-center'>
+              <span className='pr-2 font-bold'>Step 2:</span>
+              <span className='pr-2'>Set up your application</span>
+            </div>
+            <div className='pl-4 mt-4'>
+              <div className="verify-account-sid">
+                <label htmlFor={'ACCOUNT_SID'}>
```

Comment:
i see multiple labels with htmlFor={`ACCOUNT_SID`}; labels should correspond 1:1 with their respective `<input />` `id`